source/Octopus.Cli/Commands/Runbooks/RunRunbookCommand.cs

```diff
@@ -197,11 +201,12 @@ protected override Task ValidateParameters()
             }
 
             if ((RunAt ?? DateTimeOffset.Now) > NoRunAfter)
-                throw new CommandException("The Run will expire before it has a chance to execute.  Please select an expiry time that occurs after the deployment is scheduled to begin");
+                throw new CommandException(
+                    "The Run will expire before it has a chance to execute.  Please select an expiry time that occurs after the deployment is scheduled to begin");
 
             CheckForIntersection(IncludedMachineIds.ToList(), ExcludedMachineIds.ToList());
 
-            if (TenantIds.Contains("*") && (TenantIds.Count > 1 || TenantTagNames.Count > 0))
+            if (TenantNamesOrIds.Contains("*") && (TenantNamesOrIds.Count > 1 || TenantTagNames.Count > 0))
```

Comment:
Just double checking - the server supports the wildcard alias already correct?

Reply:
Last I checked (on friday). I'll check again right now.

Reply:
Shizzy, thats, right -- this needs to be handled in the CLI now -- good catch good catch.

Reply:
Well we need to pass through the wildcard if we get it, and not attempt to retrieve anything by name or id :)

Reply:
👍 Should be all set now. I added a method to apply this logic in the cli.